apps/mobile-wallet/src/features/settings/WalletDeleteModal.tsx

```diff
@@ -66,38 +67,44 @@ const WalletDeleteModal = ({ onDelete, ...props }: WalletDeleteModalProps) => {
   }
 
   return (
-    <>
-      <ModalContent verticalGap {...props}>
-        <ScreenSection>
-          <ModalScreenTitle>⚠️ {t('Delete "{{ walletName }}"?', { walletName })}</ModalScreenTitle>
-        </ScreenSection>
-        <ScreenSection>
-          <AppText color="secondary" size={18}>
-            {t('Do you really want to delete this wallet from your device?')}
-          </AppText>
-          <AppText color="secondary" size={18}>
-            {t('You can always restore it later using your secret recovery phrase.')}
-          </AppText>
-          <AppText color="secondary" size={18}>
-            {t('If so, please enter the wallet name below, and hit the delete button.')}
-          </AppText>
-        </ScreenSection>
-        <ScreenSection>
-          <Input label={t('Wallet name')} value={inputWalletName} onChangeText={setInputWalletName} />
-        </ScreenSection>
-        <ScreenSection>
-          <Button
-            title={t('Delete')}
-            variant="alert"
-            onPress={handleDeleteConfirmPress}
-            disabled={inputWalletName !== walletName}
-            iconProps={{ name: 'trash' }}
-          />
-        </ScreenSection>
-      </ModalContent>
-      <SpinnerModal isActive={isLoading} />
-    </>
+    <BottomModal
+      id={id}
+      maximisedContent={Platform.OS === 'ios'}
```

Comment:
I am not 100% sure this is needed anymore, I think we recently pushed a fix that pushes the input fields higher when they keyboard is open. to be checked

Reply:
Will check in a follow up PR. Adding a comment for now.